docs/en/interfaces/cli.md

```diff
@@ -67,22 +67,22 @@ The command-line client allows passing external data (external temporary tables)
 
 ### Queries with Parameters {#cli-queries-with-parameters}
 
-You can create a query with parameters, and pass values for these parameters with the parameters of the client app. For example:
+You can create a query with parameters and pass values of the corresponding client app parameters to them. For example:
 
 ```bash
 clickhouse-client --param_parName="[1, 2]"  -q "SELECT * FROM table WHERE a = {parName:Array(UInt16)}"
 ```
 
-#### Syntax of a Query {#cli-queries-with-parameters-syntax}
+#### Query Syntax {#cli-queries-with-parameters-syntax}
 
-Format a query by the standard method. Values that you want to put into the query from the app parameters place in braces and format as follows:
+Format a query using the standard method. Place the values that you want to pass from the app parameters to the query in braces in the following format:
```

Comment:
what does "standard method" mean? this syntax has nothing to do with SQL standard